security/deprecated-blowfish/deprecated-blowfish.clj

```diff
@@ -0,0 +1,8 @@
+(ns deprecated-blowfish
+  (:import [javax.crypto Cipher]))
+
+(defn vulnerable []
+  (Cipher/getInstance "Blowfish/CBC/PKCS5Padding"))
+
+(defn not-vulnerable []
+  (Cipher/getInstance "RSA/ECB/PKCS1Padding"))
```

Comment:
```suggestion
  (Cipher/getInstance "RSA/CBC/PKCS1Padding"))
```
Just to ensure this rule doesn't clash with the one testing for ECB mode of operation.